frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.test.ts

```diff
@@ -167,7 +167,9 @@ describe('filtersToQueryNode', () => {
             const query: InsightQueryNode = {
                 kind: NodeKind.RetentionQuery,
                 filterTestAccounts: true,
+                retentionFilter: {},
             }
+            result.retentionFilter = {} // Put this here to make TS happy
```

Comment:
It seems to make TS unhappy...
I don't understand why we need it. Shouldn't `filtersToQueryNode` take care of stuff like that?

Reply:
I have the query typed to require `retentionFilter`, hence added it, but since it's empty the result var didn't contain it and the equal test was failing. Anyway, I just used a cast to bypass issues now.